de.dlr.sc.virsat.model.extension.cefx.ui/src/de/dlr/sc/virsat/model/extension/cefx/ui/templates/DLRCEFXStudyCommandHelper.java

```diff
@@ -0,0 +1,175 @@
+import de.dlr.sc.virsat.model.extension.cefx.model.EquipmentPowerParameters;
+import de.dlr.sc.virsat.model.extension.cefx.model.EquipmentTemperatureParameters;
+import de.dlr.sc.virsat.model.extension.cefx.model.SubSystemMassParameters;
+import de.dlr.sc.virsat.model.extension.cefx.model.SubSystemPowerParameters;
+import de.dlr.sc.virsat.model.extension.cefx.model.SystemMassParameters;
+import de.dlr.sc.virsat.model.extension.cefx.model.SystemParameters;
+import de.dlr.sc.virsat.model.extension.cefx.model.SystemPowerParameters;
+import de.dlr.sc.virsat.model.extension.ps.model.ConfigurationTree;
+import de.dlr.sc.virsat.model.extension.ps.model.ElementConfiguration;
+import de.dlr.sc.virsat.project.editingDomain.VirSatTransactionalEditingDomain;
+import de.dlr.sc.virsat.project.resources.command.CreateSeiResourceAndFileCommand;
+/**
+ * Set the parameter for the Concepts (System, SubSystem, Equipment)
+ *
+ */
+public class DLRCEFXStudyCommandHelper {
+	
+	/**
+	 * private Constructor
+	 */
+	private DLRCEFXStudyCommandHelper() {
+	}
+	/**
+	 * Get the cefx concept from the domain
+	 * @param domain the domain
+	 * @return the cefx concept
+	 */
+	public static Concept getCefxConcept(VirSatTransactionalEditingDomain domain) {
+		Repository currentRepository = domain.getResourceSet().getRepository();
+		ActiveConceptHelper acHelper = new ActiveConceptHelper(currentRepository);
+		Concept activeConcept = acHelper.getConcept(Activator.PLUGIN_ID);
+		return activeConcept;
+	}
+
+	public static Concept getPsConcept(VirSatTransactionalEditingDomain domain) {
+		Repository currentRepository = domain.getResourceSet().getRepository();
+		ActiveConceptHelper acHelper = new ActiveConceptHelper(currentRepository);
+		Concept activeConcept = acHelper.getConcept(de.dlr.sc.virsat.model.extension.ps.Activator.getPluginId());
+		return activeConcept;
+	}
+	/**
+	 * 
+	 * @param conceptPs
+	 * @return create the system as the configuration tree.
+	 */
+	public static ConfigurationTree createSystemAsConfigurationTree(Concept conceptPs) {
+		
+		ConfigurationTree system = new ConfigurationTree(conceptPs);
+		system.setName("System");
+		return system;
+	}
+	
+	/**
+	 * 
+	 * @param conceptPs
+	 * @return create Sub-system as element configuration
+	 */
+	public static ElementConfiguration createSubSystemAsElementConfiguration(Concept conceptPs) {
+		ElementConfiguration subSystem = new ElementConfiguration(conceptPs);
+		subSystem.setName("subSystem");
+		return subSystem;
+	}
+
+	/**
+	 * @param conceptPs
+	 * @return create an equipment as element configuration.
+	 */
+	public static ElementConfiguration createEquipmentAsElementConfiguration(Concept conceptPs) {
+		ElementConfiguration equipment = new ElementConfiguration(conceptPs);
+		equipment.setName("equipment");
+		return equipment;
+	}
+
+	//CHECKSTYLE:OFF
+	public static void addSystemParameters(Concept conceptCEFX, ConfigurationTree system) {
```

Comment:
Please don't turn of Checkstyle here... It would be better to define the magic numbers as constant in the class... Like DEFAULT_SYSTEM_MAGRIN and so on... Then it's better to change later

Reply:
Nice 👍 Then please also remove the //CHECKSTYLE:OFF comment here ;)

Reply:
I can not remove it.